src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java

```diff
@@ -75,7 +75,7 @@
     private String language;
     private String programToken;
     private String timeZone;
-    private List<HyperwalletDocument> documents;
+    private List<HyperwalletVerificationDocument> documents;
```

Comment:
I'm assuming we have no clients using this version yet. Is that correct? otherwise, it becomes a breaking change

Reply:
Checking with Willem on this